lighthouse-core/gather/gather-runner.js

```diff
@@ -313,7 +333,7 @@ class GatherRunner {
     }
 
     // Resolve on tracing data using passName from config.
-    return passData;
+    return [passData, pageLoadError];
```

Comment:
thoughts?

Reply:
WDYT about attaching it to `passData` instead? It's a relatively grab-bag object of what a happened during a pass, so it makes some sense :)

Reply:
Sure. I'll add it as an optional property. I avoided it in the first place b/c I didn't want to modify all usages of this type, but I completely forgot about optional properties :P